resources/js/layouts/guest-layout.tsx

```
@@ -15,18 +21,49 @@ export default function GuestLayout({
   children,
 }: PropsWithChildren<GuestLayoutProps>) {
   return (
-    <div className="flex min-h-screen flex-col items-center px-4 pt-6 sm:justify-center sm:pt-0">
-      <Flash />
-      <Link href="/" aria-label="Go to homepage">
-        <Logo className="size-10" />
-      </Link>
+    <div className="min-h-screen flex flex-col lg:flex-row">
```

Biome (2.4.12)

[error] These CSS classes should be sorted.

(lint/nursery/useSortedClasses)